.github/workflows/test.yml

```diff
@@ -9,7 +9,11 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
+      - name: Use Node.js 20
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'      
```

Comment:
```suggestion
      - uses: actions/setup-node@v4
        with:
          node-version-file: package.json
```

Reply:
this uses the newly added `engines` value, so the node version won't need to be updated here in the future